src/walmart-msw.js

```diff
@@ -1,37 +1,38 @@
-
 require('./global');
 const axios = require('axios');
 let httpClient = null;
 
-module.exports = function(clientId, secretKey, host, version, name){
-
-    //Initialize httpClient custom Axios Instance for WMS
+module.exports = function (clientId, secretKey, version, name) { // Initialize httpClient custom Axios Instance for WMS
     httpClient = axios.create({
-        baseURL: host,
+        baseURL: 'https://marketplace.walmartapis.com/',
```

Comment:
@pedrohma  Because you use the marketplace directly here?

Reply:
Because the url is a constant, but the version is not. Agree?

Reply:
Ok you are ok

Reply:
Yes you are ok, but the package can be used with the sandbox mode, and in this case, the URL is for production hardcode.

Reply:
Agreeded @drbarzaga. What do you think about [this](https://github.com/drbarzaga/walmart-mws/pull/1/commits/97a5f218faff0e85703ab972c98df37513026d10)?

Reply:
Cool a good solution!